README.md

```diff
@@ -12,6 +12,13 @@ This repository contains a set of JSON objects that implementers of JSON Schema
 It is meant to be language agnostic and should require only a JSON parser.
 The conversion of the JSON objects into tests within a specific language and test framework of choice is left to be done by the validator implementer.
 
+## Live at Head
+
+JSON Schema Test Suit now follows the
```

Comment:
Nothing has changed here -- this has always been the case, so I also wouldn't say "now".